### PR TITLE
docs: add entries for cli smtp config #27374

### PR DESCRIPTION
**Pull request by rhnaxifg4y**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

NO

## What kind of change does this PR introduce?

docs update, ...

## What is the current behavior?

https://web.archive.org/web/20240425063607/https://supabase.com/docs/guides/cli/config

## What is the new behavior?

https://github.com/supabase/cli/pull/2427

## Additional context

https://github.com/supabase/supabase/pull/27374

### CODE REVIEW

**Comment by charislam** [comment]
@sweatybridge, could you double-check the new options please? 🙏🏼

**Review by sweatybridge — APPROVED** [review]
Thanks a lot of updating these. I should have approved sooner.

File: apps/docs/spec/cli_v1_config.yaml [hunk 1]
```diff
@@ -443,6 +443,54 @@ parameters:
       - name: 'Auth Server configuration'
         link: 'https://supabase.com/docs/reference/auth'
 
+  - id: 'auth.email.smtp.host'
+    title: 'auth.email.smtp.host'
+    tags: ['auth']
+    required: true
```
Comment (sweatybridge) [inline apps/docs/spec/cli_v1_config.yaml:449]:
```suggestion
    required: false
```

File: apps/docs/spec/cli_v1_config.yaml [hunk 2]
```diff
@@ -443,6 +443,54 @@ parameters:
...
+    default: 'inbucket'
+    description: |
+      Hostname or IP address of the SMTP server.
+
+  - id: 'auth.email.smtp.port'
+    title: 'auth.email.smtp.port'
+    tags: ['auth']
+    required: true
```
Comment (sweatybridge) [inline apps/docs/spec/cli_v1_config.yaml:457]:
```suggestion
    required: false
```

File: apps/docs/spec/cli_v1_config.yaml [hunk 3]
```diff
@@ -443,6 +443,54 @@ parameters:
...
+    default: 2500
+    description: |
+      Port number of the SMTP server.
+
+  - id: 'auth.email.smtp.user'
+    title: 'auth.email.smtp.user'
+    tags: ['auth']
+    required: false
+    default: 'None'
```
Comment (sweatybridge) [inline apps/docs/spec/cli_v1_config.yaml:466]:
```suggestion
```

File: apps/docs/spec/cli_v1_config.yaml [hunk 4]
```diff
@@ -443,6 +443,54 @@ parameters:
...
+    description: |
+      Username for authenticating with the SMTP server.
+
+  - id: 'auth.email.smtp.pass'
+    title: 'auth.email.smtp.pass'
+    tags: ['auth']
+    required: false
+    default: 'None'
```
Comment (sweatybridge) [inline apps/docs/spec/cli_v1_config.yaml:474]:
```suggestion
```

File: apps/docs/spec/cli_v1_config.yaml [hunk 5]
```diff
@@ -443,6 +443,54 @@ parameters:
...
+    description: |
+      Password for authenticating with the SMTP server.
+
+  - id: 'auth.email.smtp.admin_email'
+    title: 'auth.email.smtp.admin_email'
+    tags: ['auth']
+    required: true
```
Comment (sweatybridge) [inline apps/docs/spec/cli_v1_config.yaml:481]:
```suggestion
    required: false
```

File: apps/docs/spec/cli_v1_config.yaml [hunk 6]
```diff
@@ -443,6 +443,54 @@ parameters:
...
+    default: 'admin@email.com'
+    description: |
+      Email used as the sender for emails sent from the application.
+
+  - id: 'auth.email.smtp.sender_name'
+    title: 'auth.email.smtp.sender_name'
+    tags: ['auth']
+    required: false
+    default: 'None'
```
Comment (sweatybridge) [inline apps/docs/spec/cli_v1_config.yaml:490]:
```suggestion
```

**Pull request merged**